### PR TITLE
Added Python coding style markup.

### PR DESCRIPTION
**Pull request by AustinSmith29**
Alrigh done adding python style guidelines. tLet me know if you want anything changed or added.

### CODE REVIEW

File: coding-guidelines/python.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+Coding Style and Guidelines
+===========================
```
Comment (mirdaki) [inline coding-guidelines/python.md]: Could you please use the '#' for headers? That would keep it consistent with other club Markdown files (I need to make a style guideline for that too).

File: coding-guidelines/python.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+```python
+from ex_module import Foo, FooBar
+```
+Avoid wildcard imports (i.e. `from <module> import *`); they pollute the
+namespace.
+
+Imports should be grouped in the following order:
+1. standard library imports
+2. related third party imports
+3. local application/library specific imports
```
Comment (mirdaki) [inline coding-guidelines/python.md]: Should the list items be capitalized?

File: coding-guidelines/python.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+x = x + 1	# Compensate for border.
+
+```
+### Documentation Strings
+Docstrings are a extremely useful form of comment and they should
+be used liberally. It is good practice to write docstrings for all
+public modules, functions, classes, and methods.
+
+Commonly called docstrings.
+
```
Comment (mirdaki) [inline coding-guidelines/python.md]: Please describe why docstrings are useful

**Review by mirdaki — CHANGES_REQUESTED** [review]
Great job on the style guidelines! There are just a few formatting changes and one clarification to make

**Comment by AustinSmith29** [comment]
Alright I made the requested changes.

**Comment by mirdaki** [comment]
Awesome! Thank you for the help

**Pull request merged**